multimedia/libvpx/Portfile

```diff
@@ -31,30 +31,21 @@ fetch.type          git
 git.url             https://chromium.googlesource.com/webm/${name}
 git.branch          v${version}
 
-# support for non-intel archs removed in version 1.5.0
-supported_archs     x86_64 i386 arm64
+# on BigSur or newer, use a recent 20210203 commit with the arm64 fixes in it
+# because we also need to support +universal, do this for all builds for now
 
-depends_build-append port:yasm
-
-patchfiles          patch-build-make-configure.sh.diff \
-                    patch-Makefile.diff
+if {${os.platform} eq "darwin" && ${os.major} >= 20} {
+    git.branch      5a4cfa95630ee3ebb5d74cd6c21336492e8707a4
+    version         1.9.0-20210203
+}
```

Comment:
I'm not 100% sure this is the cleanest way to do this.

we could also just change the git.branch as I did here, leave the version as 1.9.0, and add a ```revision 1``` if that would be considered more neat and tidy. 

SImple to do.